test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -45,6 +45,49 @@ export class SetupEnvironmentHelper {
         SetupEnvironmentHelper.patchMetroConfig(workspacePath);
     }
 
+    public static  prepareHermesReactNativeApplication(workspaceFilePath: string, resourcesPath: string, workspacePath: string, appName: string, customEntryPointFolder: string, version?: string) {
+        let commandClean = path.join(workspacePath, "android", "gradlew") + " clean";
+        let commandBuild = path.join(workspacePath, "android", "gradlew") + " build";
+
+        console.log(`*** Executing  ${commandClean}...`);
+        cp.execSync(commandClean, { cwd: path.join(workspacePath, "android"), stdio: "inherit" });
+
+        let customEntryPointFile = path.join(resourcesPath, customEntryPointFolder, "App.js");
+        let launchConfigFile = path.join(resourcesPath, "launch.json");
+        let vsCodeConfigPath = path.join(workspacePath, ".vscode");
+        let appGradlBuildFilePath = path.join(workspacePath, "android", "app", "build.gradle");
```

Comment:
Can we move operations with Gradle files in a new function?

Reply:
Created a new function for copying Gradle files.